install.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+set -Eeuo pipefail
+
+PORT=8321
+OLLAMA_PORT=11434
+MODEL_ALIAS="llama3.2:3b"
+SERVER_IMAGE="llamastack/distribution-ollama:0.2.2"
+WAIT_TIMEOUT=300
+
+log(){ printf "\e[1;32m%s\e[0m\n" "$*"; }
+die(){ printf "\e[1;31m❌ %s\e[0m\n" "$*" >&2; exit 1; }
+
+if command -v docker &> /dev/null; then
+  ENGINE="docker"
+  HOST_DNS="host.docker.internal"
+elif command -v podman &> /dev/null; then
+  ENGINE="podman"
+  HOST_DNS="host.containers.internal"
+else
+  die "Docker or Podman is required. Install Docker: https://docs.docker.com/get-docker/ or Podman: https://podman.io/getting-started/installation"
+fi
+
+# Clean up any leftovers from earlier runs
+for name in ollama-server llama-stack; do
+  # detect existing containers
+  ids=$($ENGINE ps -aq --filter "name=^${name}$")
+  if [ -n "$ids" ]; then
+    log "⚠️   Found existing container(s) for '${name}', removing..."
+    $ENGINE rm -f "$ids"
+  fi
+done
+
+###############################################################################
+# 1. Ollama
+###############################################################################
+log "🦙  Starting Ollama…"
+$ENGINE run -d --name ollama-server \
+  -p "${OLLAMA_PORT}:11434" \
+  -v ollama-models:/root/.ollama \
+  ollama/ollama >/dev/null
```

Comment:
```suggestion
  ollama/ollama > /dev/null 2>&1
```

Somehow, 2 prints out a lot of logs, let's not pollute the output.